src/app/GitUI/GitUICommands.cs

```diff
@@ -749,10 +750,16 @@ bool Action()
             }
         }
 
-        private bool StartResetChangesDialog(string fileName)
+        private bool StartResetChangesDialog(string[] names)
```

Comment:
```suggestion
        private bool StartResetChangesDialog(string[] fileNames)
```

Are these just names, relative or absolute file names?

Reply:
> Are these just names, relative or absolute file names?

It is what the user (or the shell extension, i.e. the Explorer) passes as command line args.
Since these can be folders, too, I have removed "file".
The first step in this function is normalization of anything to relative paths.

Reply:
Consider adding a comment